templates/sentry/4/rancher-compose.yml

```diff
@@ -0,0 +1,137 @@
+version: '2'
+catalog:
+  name: Sentry
+  version: 8.21.0
```

Comment:
Package version field should be different from previous releases. 
Could you please change it??
```
-  version: 8.21.0
+  version: 8.21.0-1
```
Also, `templates/sentry/config.yml` version field should be updated.